<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>나눔고딕</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>GPA Management System</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>700</width>
     <height>500</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="titleLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>400</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>나눔고딕</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>중앙대학교 평점 관리 시스템</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame2">
    <property name="geometry">
     <rect>
      <x>569</x>
      <y>410</y>
      <width>81</width>
      <height>40</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="githubButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>60</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕</family>
      </font>
     </property>
     <property name="toolTip">
      <string>새 창 열기</string>
     </property>
     <property name="text">
      <string>GitHub</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>600</width>
      <height>261</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>115</y>
       <width>450</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕</family>
      </font>
     </property>
     <property name="text">
      <string>종합 평점 분석 및 목표 설정</string>
     </property>
    </widget>
    <widget class="QPushButton" name="to_firstButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕</family>
      </font>
     </property>
     <property name="text">
      <string>기본정보 등록</string>
     </property>
    </widget>
    <widget class="QPushButton" name="to_secondButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>100</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕</family>
      </font>
     </property>
     <property name="text">
      <string>학기별 평점</string>
     </property>
    </widget>
    <widget class="QLabel" name="label2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>65</y>
       <width>450</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕</family>
      </font>
     </property>
     <property name="text">
      <string>학기별 평점 분석</string>
     </property>
    </widget>
    <widget class="QLabel" name="label1">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>15</y>
       <width>450</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕</family>
      </font>
     </property>
     <property name="text">
      <string>전공 및 성적 내역 등록</string>
     </property>
    </widget>
    <widget class="QPushButton" name="to_thirdButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>100</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕</family>
      </font>
     </property>
     <property name="text">
      <string>종합 평점</string>
     </property>
    </widget>
    <widget class="QLabel" name="label4">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>165</y>
       <width>450</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕</family>
      </font>
     </property>
     <property name="text">
      <string>졸업기준 중 이수학점 확인</string>
     </property>
    </widget>
    <widget class="QPushButton" name="to_fourthButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>100</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕</family>
      </font>
     </property>
     <property name="text">
      <string>이수학점내역</string>
     </property>
    </widget>
    <widget class="QPushButton" name="to_fifthButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>100</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕</family>
      </font>
     </property>
     <property name="text">
      <string>도움말</string>
     </property>
    </widget>
    <widget class="QLabel" name="label5">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>215</y>
       <width>450</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕</family>
      </font>
     </property>
     <property name="text">
      <string>프로그램 이용 도움말</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
